contracts/apps/capital/CapitalApp.sol

```diff
@@ -38,7 +37,7 @@ contract CapitalApp is Application {
     reverseSales[_saleAddress] = saleIndex;
     saleIndex += 1;
 
-    address tknAddr = TokensOrgan(dao).getToken(sale.tokenId());
+    address tknAddr = address(0);
```

Comment:
Capital app is going away with my next PR that adds token sales into ownership.

I just changed it to 0 so it would compile after removing tokensorgan